sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/ParentBasedSampler.java

```diff
@@ -19,7 +19,7 @@
 /**
  * A Sampler that uses the sampled flag of the parent Span, if present. If the span has no parent,
  * this Sampler will use the "root" sampler that it is built with. See documentation on the {@link
- * Builder} methods for the details on the various configurable options.
+ * ParentBasedSamplerBuilder} methods for the details on the various configurable options.
  */
 @Immutable
 public class ParentBasedSampler implements Sampler {
```

Comment:
can we make this class non-public now? That would be an improvement, if we can.

Reply:
Also final :). Cannot be non-public if the "builder" method is on this class.

Reply:
yeah, I realized that and forgot to remove the comment.

Reply:
Nice, thanks!